sentence.go

```diff
@@ -28,6 +28,16 @@ var (
 // ParserFunc callback used to parse specific sentence variants
 type ParserFunc func(BaseSentence) (Sentence, error)
 
+// ParseNotSupportedError is returned when parsed sentence is not supported
+type ParseNotSupportedError struct {
```

Comment:
Call this `NotSupportedError`.

Reply:
I have renamed it to `ErrNotSupported` so it would be like most of the standard library errors.

Reply:
The `ErrName` convention is only used when exporting an error value. For example:

``` go
var ErrNotSupported = errors.New("name: prefix not supported")
```
However, the `NameError` convention (like you originally had) is used for error types.

Reply:
seems, changed it.